restore-cognito.md

```diff
@@ -0,0 +1,30 @@
+#Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+###Required IAM permissions
+To be able to do the user restoration you need to be in the engineering group in AWS. If you are a developer you should already have those permissions.  If you need to be added to the permission group, speak with Iva or Pol, who will add it through IAM.
+
+**Downloading user data**
+Here do either 1 or 2 depending on the situation.
+
+1. In case there is no cognito user pool or the user pool is empty 
+We have weekly user backup.  You will need to find the latest users backup file you need to go to biomage-backups-production-{accoundId} S3 bucket and find the latest object in the bucket. The objects names are the date they are created so using this you can find the latest one.
```

Comment:
`You will need to find the latest users backup file you need to go to ...` will benefit some rephrasing. Better have it as `You will need to find the latest users backup file. To do this, go to biomage-backups-production-{accountId} ...`